src/Kdyby/Translation/CatalogueCompiler.php

```diff
@@ -60,10 +67,22 @@ public function enableDebugMode()
 	}
 
 
+	/**
+	 * Replaces cache storage with simple memory storage (per-request).
+	 */
+	public function disableDebugMode()
```

Comment:
ano, potřebuji to tam. 
Když není zapnutý debug mód, tak TranslationWriter nemá načtené resources.
Tedy musím zavolat enableDebugMode, aby měl TranslatonWriter resources, do kterých může zapsat. 
Jenže pak se změna neprojeví u cache, jelikož mám zapnutou MemoryStorage a pro ostatní uživatele je stále klasickáPhpArrayStorage. Takže pokud chci, aby se automaticky projevily překlady i bez debug mode, musím těsně před zápisem přehodit zpět na persistentní storage (právě metodou disableDebugMode). Pak se překlady v celé aplikaci projeví hned po změně a nemusím kvůli tomu mazat cache celé aplikace.